docs/contributing.md

```diff
@@ -0,0 +1,174 @@
+# Contributing to RMF
+Contributions via pull requests to the various RMF-related repositories are welcome. In general, we expect contributors to follow the ROS 2 [contributing guidelines](https://index.ros.org/doc/ros2/Contributing/) with exception of the code style. The RMF codebases adopt slightly different guidelines from the ROS 2 [style guide](https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/#codestyle). The objective of this document is to highlight these guidelines for the following languages
```

Comment:
```suggestion

Contributions via pull requests to the various RMF-related repositories are welcome.
In general, we expect contributors to follow the [ROS 2 contributing guidelines](https://index.ros.org/doc/ros2/Contributing/), with the exception of code style.
The RMF code adopts slightly different guidelines from the [ROS 2 style guide](https://index.ros.org/doc/ros2/Contributing/Code-Style-Language-Versions/#codestyle).
The objective of this document is to provide the RMF code style guidelines for the following languages.
```

One line per sentence with no wrapping makes doing diffs in the future much easier.
(Also there are some grammatical changes in the above suggestion.)

Reply:
The code style guide below is not a full-coverage style guide. Should we say "Follow the ROS 2 style guide except for the modifications listed below." or something along those lines?